nixos/modules/services/misc/mediatomb.nix

```
@@ -52,7 +51,7 @@ let
           <accept-url>yes</accept-url>
           <first-resource>yes</first-resource>
           <accept-ogg-theora>yes</accept-ogg-theora>
-          <agent command="${libsForQt5.vlc}/bin/vlc"
+          <agent command="${vlc}/bin/vlc"
```

We should set `mainProgram` on `vlc` and use

```suggestion
          <agent command="${lib.getExe vlc}"
```